### ISSUE TITLE
Why does training `deepseek-ditstill-qwen-7b` with `openthinker-1` lead to a drop in AIME to ~0.4?

### ISSUE THREAD

**Issue by NonvolatileMemory** [open-thoughts/open-thoughts#26]
Hi team,

Thank you for releasing `openthinker-1` and `deepseek-ditstill-qwen-7b` — they’re very exciting projects.

I’m currently training `deepseek-ditstill-qwen-7b` using the `openthinker-1` trainer. However, I observed that the AIME score dropped significantly to around **0.4**, which is much lower than expected.

Could you please help me understand:

- Whether this behavior is expected?
- If there are any known compatibility issues or caveats when using `openthinker-1` to train the `deepseek-ditstill-qwen-7b` model?
- Any recommended training settings or initialization conditions to avoid this degradation?

Looking forward to your guidance. And the lr is 1e-5, is that the reason?

Best regards,  
Cunxiao

**Comment by RulinShao** [comment]
I observed the same phenomena. Curious about the reason, too :)

**Comment by RyanMarten** [comment]
We trained OpenThinker-7B on Qwen-2.5-7B-Instruct with the [following hyperparameters](https://huggingface.co/open-thoughts/OpenThinker-7B#training-hyperparameters):

> learning_rate: 1e-05
> train_batch_size: 1
> eval_batch_size: 8
> seed: 42
> distributed_type: multi-GPU
> num_devices: 32
> gradient_accumulation_steps: 3
> total_train_batch_size: 96
> total_eval_batch_size: 256
> optimizer: Use OptimizerNames.ADAMW_TORCH with betas=(0.9,0.999) and epsilon=1e-08 and optimizer_args=No additional optimizer arguments
> lr_scheduler_type: cosine
> lr_scheduler_warmup_ratio: 0.1
> num_epochs: 3.0

It is interesting you observe this behavior when further training the DeepSeek-R1-Distill-7B model on OpenThoughts. We did not train on top of the distill model, so unfortunately don't have any additional numbers for you to compare with.

**Comment by EtashGuha** [comment]
You may have to play with the hyperparameters a touch. Maybe the learning rate is too high, or you could use more LR warmup. Let us know what you find!
